utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/SparkTestBase.scala

```diff
@@ -17,17 +17,69 @@ package za.co.absa.enceladus.utils.testUtils
 
 import org.apache.log4j.{Level, Logger}
 import org.apache.spark.sql.SparkSession
+import org.apache.spark.SparkConf
+import com.typesafe.config.ConfigFactory
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{Path, FileSystem}
+import scala.collection.JavaConversions._
+import java.io.File
 import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
 
-trait SparkTestBase {
+trait SparkTestBase { self =>
   System.setProperty("user.timezone", "UTC");
```

Comment:
I know it is not your code, but could you delete the `;` here?

Reply:
👍